modules/core/test/GVL.test.ts

```diff
@@ -7,30 +7,32 @@ import {XMLHttpTestTools} from '@iabtcf/testing';
 import {Json} from '../src/Json';
 
 // eslint-disable-next-line @typescript-eslint/no-var-requires
-const vendorlistJson = require('@iabtcf/testing/lib/vendorlist/vendor-list-v24.json');
+const vendorlistJson = require('@iabtcf/testing/lib/vendorlist/v2/vendor-list-v24.json');
 // eslint-disable-next-line @typescript-eslint/no-var-requires
-const translationJson = require('@iabtcf/testing/lib/vendorlist/purposes-fr.json');
+const translationJson = require('@iabtcf/testing/lib/vendorlist/v2/purposes-fr.json');
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const vendorlistJson22 = require('@iabtcf/testing/lib/vendorlist/v2.2/vendor-list.json');
```

Comment:
The namespace is changed from @iabtcf to @iabtechlabtcf

Reply:
The same thing in TCModel.test.ts and in model/PurposeRestrictionVector.test.ts

Reply:
You are right we need to update this PR because of library location has been changed https://www.npmjs.com/package/@iabtechlabtcf